doc/setup.md

```diff
@@ -0,0 +1,13 @@
+# Setting up enclave-enabled EC2 host
+
+1. Go to `Launch an Instance` in the EC2 console
+2. Select `Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type` under AMI
```

Comment:
I would say rather that most recent Linux distros can work, but installing the tools is easier with Amazon Linux 2023 and Amazon Linux 2.

Reply:
I agree this is a better recommendation